hangman/game.py

```diff
@@ -1,13 +1,101 @@
 from .exceptions import *
+import random
 
 
 class GuessAttempt(object):
-    pass
-
+    new_word = ""
+    def __init__(self, guess, hit=False, miss=False):
+        self.hit = hit
+        self.miss = miss
+        self.guess = guess
+        if self.hit == True and self.miss== True:
+            raise InvalidGuessAttempt
+        
+            
+    def is_hit(self):
+        return self.hit
+        
+    def is_miss(self):
+        return self.miss
 
 class GuessWord(object):
-    pass
+    def __init__(self, answer):
+        self.answer = answer.lower()
+        self.masked = '*' * len(self.answer)
+        if not answer:
```

Comment:
"Personally" I like to put these things at the beginning of the method. It's like "Design by contract". Specifying the "pre-conditions" of the method. Purely subjective, tho.